mobile/src/navigations/root/root.tsx

```diff
@@ -56,8 +59,20 @@ const Root: React.FC = () => {
         if (!currentUserData) {
             return;
         }
+        const nextStep = getNextStep(completedStep);
         void dispatch(getUserDetails({ userId: currentUserData.id }));
-    }, [currentUserData, currentUserData?.id, dispatch]);
+
+        if (nextStep && !isProfileComplete) {
+            navigate(nextStep);
```

Comment:
Here I get an error from TS. 
`No overload matches this call.
  Argument of type '[string]' is not assignable to parameter of type 'never'.
  Overload 2 of 2, '(options: never): void', gave the following error.
    Argument of type 'string' is not assignable to parameter of type 'never'.`

Reply:
Let's not navigate, let's set initial route in options depending on last completes step

Reply:
I tried to pass this nextStep to the talent onboarding, but I can't pass it without navigate. And when I use navigate I receive the same error. Maybe it'd be better if I just use type assertion `nextStep as never`?

Reply:
You can access store in onboarding nav

Reply:
I made it, when I log nextStep it logs it, but initialROuteName stays as it was

Reply:
hmm then... let's add not `navigate`, but `reset` in **talent navigator**

Reply:
had to delete isPendingOnboardingData in the root because with it, onboarding is not working properly